node/hubController.js

```diff
@@ -326,7 +327,7 @@ class HubController {
     }
 
     _createTranslator(translatorName, deviceInfo) {
-        this.ConsoleLogger.info(`_createTranslator ${translatorName}`);
+        this.logger.info(`_createTranslator ${translatorName}`);
         return this.OpenT2T.createTranslatorAsync(translatorName, deviceInfo).then( translator => {
```

Comment:
From discussing with Dave, that's the plan, yes.  Is the idea to pass in an OpenT2T logger configured how we want, or just the custom transport we want?